src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -23,10 +24,12 @@
 class DumpExtension extends \Twig_Extension
 {
     private $cloner;
+    private $dumper;
 
-    public function __construct(ClonerInterface $cloner)
+    public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper = null)
```

Comment:
It looks like `AbstractDumper::dump` already has an optional `output` parameter. Could this be moved to `DataDumperInterface` without breaking BC? Or maybe a different interface with  `dumpTo(Data $data, $output)` ?